api/overview/azure/latest/search-index.md

```diff
@@ -1,4 +1,4 @@
 | Reference | Package | Source |
 |---|---|---|
-|[Cognitive Search](search.documents-readme.md)|[Azure.Search.Documents](https://www.nuget.org/packages/Azure.Search.Documents)|[GitHub](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/search/Azure.Search.Documents)|
-|[Resource Management - Cognitive Search](resourcemanager.search-readme.md)|[Azure.ResourceManager.Search](https://www.nuget.org/packages/Azure.ResourceManager.Search)|[GitHub](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/search/Azure.ResourceManager.Search)|
+|[Azure AI Search](search.documents-readme.md)|[Azure.Search.Documents](https://www.nuget.org/packages/Azure.Search.Documents)|[GitHub](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/search/Azure.Search.Documents)|

```

Comment:
These changes will likely be overwritten by automation in the next pipeline run. That doesn't affect the merge-ability of this PR. Just FYI.